pkg/model/iam.go

```diff
@@ -48,6 +49,20 @@ const RolePolicyTemplate = `{
   ]
 }`
 
+const ServiceAccountRolePolicyTemplate = `{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": { "Federated": "{{ FederatedPrincipal }}" },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": { "{{OIDCProvider}}:sub": "{{ OIDCSub }}" }
+      }
+    }
+  ]
+}`
```

Comment:
Done for ServiceAccount roles.  For Node roles it would change the whitespace for existing cluster policies, so I left it using the string method - but I removed the templating as it's not really needed.